src/main/java/com/strandls/cca/pojo/CCAData.java

```diff
@@ -221,4 +232,20 @@ public void translate(CCATemplate template) {
 		}
 	}
 
+	public String getusername(UserServiceApi userService,Set<String> users) {
+		String userNames ="";
+		String separator = "";
+		for (String user : users) {
+			User userDetails;
+			try {
+				userDetails = userService.getUser(user);
```

Comment:
Pls check whether the getUser function wrk without passing the token, otherwise LGTM